lib/paper_trail/has_paper_trail.rb

```diff
@@ -8,6 +9,7 @@ def self.included(base)
     end
 
     module ClassMethods
+      include Callbacks
```

Comment:
I'd rather keep the contents of `Callbacks` here in `ClassMethods`. I generally agree with Steve Klabnik's article "[Mixins: A refactoring anti-pattern](http://blog.steveklabnik.com/posts/2012-05-07-mixins--a-refactoring-anti-pattern)".